appveyor.yml

```diff
@@ -92,17 +101,38 @@
                     git submodule update --init --recursive thirdparty\grpc
                 }
 
-            }
+                $boostComponents = (
+                    'boost_chrono',
+                    'boost_date_time',
+                    'boost_thread',
+                    'boost_system',
+                    'boost_unit_test_framework')
+
+                if ($env:BOND_BUILD -eq 'Python') {
+                    $boostComponents += 'boost_python'
+                }
 
-            if ($env:BOND_BOOST -eq 56) {
-                # Hard-coded Boost path per https://www.appveyor.com/docs/installed-software#languages-libraries-frameworks
+                $vsNum = tools\ci-scripts\windows\ConvertTo-MsvcVersion.ps1 -ImageName $env:BOND_VS_VERSION -Format VsNum
```

Comment:
A minor thing, but I think this is `VcNum` not a `VsNum`.

Reply:
Hmmm... Let me check. I've pushed a fixup for some of your other comments, and will come back to this one when I have an answer.